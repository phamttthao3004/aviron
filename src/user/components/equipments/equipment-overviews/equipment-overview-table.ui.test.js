import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import EquipmentOverviewTableUI from './equipment-overview-table.ui';

describe('Equipment Overview Table UI', () => {
    it('Render Equipment Overview Table UI component', () => {
        const props = { data: undefined, setOpenEdit: undefined, onChange: undefined, onSearch: undefined };
        const com = shallow(<EquipmentOverviewTableUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
