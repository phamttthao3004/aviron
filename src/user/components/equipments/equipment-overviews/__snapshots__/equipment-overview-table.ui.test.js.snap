// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Equipment Overview Table UI Render Equipment Overview Table UI component 1`] = `
<CmsTable
  className="container"
  data={Array []}
  mode="edit"
  onChange={[Function]}
  onSearch={[Function]}
  total={0}
>
  <div
    className="cms-table-content"
  >
    <table
      className="table aviron-table action-table call-data-table"
    >
      <thead>
        <tr>
          <th>
            #
          </th>
          <th>
            Machine id
          </th>
          <th>
            Equipment name
          </th>
          <th>
            Model
          </th>
          <th>
            Last online
          </th>
          <th>
            App version
          </th>
          <th>
            OS version
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</CmsTable>
`;
